web/client/components/geostory/navigation/Navigation.jsx

```diff
@@ -61,6 +67,29 @@ export default ({
         || isToolbarEnabled
         || isHomeButtonEnabled;
 
+    useEffect(() => {
+        updateUrlOnScroll(enableScrollOnLoad);
+    }, []);
+
+    useEffect(() => {
+        if (
```

Comment:
@allyoucanmap yes, see at the issue description [here](https://github.com/geosolutions-it/MapStore2-C039/issues/190)
>We can consider on automatically update the story URL with that parameter as soon the user start using the NavBar (i.e. he clicks on an item in navbar to scroll to a section). **Also the Share tool can automatically share the story at the current visualized section or it can has an option for this.**

and at the comment [here](https://github.com/geosolutions-it/MapStore2-C039/issues/190#issuecomment-656767372)

Reply:
@allyoucanmap If we remove the `GeoStoryNavigation` from the localConfig the `scrollTo` workflow is automaically removed from the application and `Include scroll position` checkbox won't be visible now

Reply:
> @allyoucanmap If we remove the GeoStoryNavigation from the localConfig the scrollTo workflow is automaically removed from the application and Include scroll position checkbox won't be visible now

This does not match the requirements.

By the @tdipisa comment this functionality should be available even if the `GeoStoryNavigation` is not in the `localConfig` so my suggestion to move this logic to a epic.